README.md

```diff
@@ -0,0 +1,176 @@
+# ROCM-CORE Introduction
+
+rocm-core is a utility which can be used to get ROCm release version. 
+It also provides the Lmod modules files for the ROCm release.
+getROCmVersion function provides the ROCm version. 
+
+Lmod module files can be loaded with the following commands.
+``` shell
+module load rocm/x.y or 
+module load rocm
+```
+
+You can find sources and binaries in our [GitHub repository](https://github.com/ROCm/rocm-core).
+
+> [!NOTE]
+> As with all ROCm projects, the documentation is open source. For more information, see [Contribute to ROCm documentation](https://rocm.docs.amd.com/en/latest/contribute/contributing.html).
+
+## Installing ROCM-CORE
+
+To install ROCM-CORE, you must first install these prerequisites:
+
+* A [ROCm](https://rocm.docs.amd.com/)-enabled platform
+
+### Installing with pre-built packages
+
+You can install ROCM-CORE on Ubuntu using
+
+```shell
+apt-get install rocm-core
+```
+
+## Building ROCM-CORE from source
+
+You can build ROCM-CORE form source
+
+First, get the sources from repository.
+
+```shell
+  git clone https://github.com/ROCm/rocm-core.git
+```
+
+Go to Root Directory, create a build directory:
+
+```shell
+  cd rocm-core; mkdir build ;  cd build
+```
+
+Next, configure CMake. Invoke cmake with the following variables define as deemed fit
+
+```shell
+cmake -DCMAKE_CURRENT_BINARY_DIR=$PWD \
+      -DCMAKE_CURRENT_SOURCE_DIR=$PWD/../ \
+      -DCMAKE_VERBOSE_MAKEFILE=1 \
+      -DCMAKE_INSTALL_PREFIX=./ \
+      -DCPACK_GENERATOR=DEB \
```

Comment:
Obviously, this is creating DEB packages.

Reply:
Thanks. Specifying DEB only is removed from the commands in the latest version